hnn_core/feed.py

```diff
@@ -5,7 +5,7 @@
 #          Christopher Bailey <bailey.cj@gmail.com>
 
 import numpy as np
-from neuron import h
+# from .utils import check_random_state
```

Comment:
this PR was originally meant to clean up ExtFeed object. Do you still want to do this in the PR or should we delegate this to another PR? If we do this in another PR, let's delete the above line and the corresponding function.

Reply:
Yeah, let's delegate this to another PR.